### ISSUE TITLE
W tensorflow/core/framework/op_kernel.cc:1745] OP_REQUIRES failed at save_restore_v2_ops.cc:222 : NOT_FOUND: Key Conv2D/L2-Loss/moving_avg not found in checkpoint

### ISSUE THREAD

**Issue by Saad-Mufti** [omidmnezami/Engagement-Recognition#14]
I am trying to run the pre-trained ER model provided as per the README file with `python ER_model.py test`.

It appears that the pretrained model TF_final is missing (or has differently named) checkpoint keys that the script expects, and I get the error in the title. 

My environment:
`Windows 11 Home, 64-bit, version 21H2`
`Python version = 3.9.5`
`tensorflow=2.8.0`
`tflearn=0.5.0`

Would someone be able to point me in the right direction? Any help would be appreciated!

**Comment by AlVedernikov** [comment]
> I am trying to run the pre-trained ER model provided as per the README file with `python ER_model.py test`.
> 
> It appears that the pretrained model TF_final is missing (or has differently named) checkpoint keys that the script expects, and I get the error in the title.
> 
> My environment: `Windows 11 Home, 64-bit, version 21H2` `Python version = 3.9.5` `tensorflow=2.8.0` `tflearn=0.5.0`
> 
> Would someone be able to point me in the right direction? Any help would be appreciated!

Hi! Where you able to solve the problem when running ER_model.py file? I am facing the same issue.

**Comment by AlVedernikov** [comment]
I was able to solve this issue. mentioned error is due to the higher version of tensofrflow. please, use these commands

conda create -n tf1_env_2 python=3.5
conda activate tf1_env_2
conda install -c conda-forge tensorflow==1.8.0
pip install numpy==1.15.2
pip install tflearn==0.3.2
pip install tflearn==0.3.2
pip install h5py==2.10.0
pip install windows-curses
pip install scipy==1.1.0
